releases/4.0.0-beta.1.md

```diff
@@ -0,0 +1,105 @@
+    + Add the `max-server-connections` configuration item (`4096` by default) to control the number of connections to a single server [#14409](https://github.com/pingcap/tidb/pull/14409)
+    + Support the isolation read specifying multiple storage engines at the server level [#14440](https://github.com/pingcap/tidb/pull/14440)
+    + Optimize the cost model of the `Apply` operator and the `Sort` operator to improve stability [#13550](https://github.com/pingcap/tidb/pull/13550) [#14708](https://github.com/pingcap/tidb/pull/14708)
+
+* TiKV
+
+    + Support fetching configuration items from the status port via HTTP API [#6480](https://github.com/tikv/tikv/pull/6480)
+    + Optimize the performance of `Chunk Encoder` in Coprocessor [#6341](https://github.com/tikv/tikv/pull/6341)
+
+* PD
+
+    + Support accessing to the distribution of hotspots in the cluster through Dashboard UI [#2086](https://github.com/pingcap/pd/pull/2086)
+    + Support capturing and displaying `START_TIME` and `UPTIME` of cluster components [#2116](https://github.com/pingcap/pd/pull/2116)
+    + Add the information of deployment path and component version in the returned message of the `member` API [#2130](https://github.com/pingcap/pd/pull/2130)
+    + Add the `component` sub-command in pd-ctl to modify and check the configuration of other components (experimental) [#2092](https://github.com/pingcap/pd/pull/2092)
+
+* TiDB Binlog
+
+    + Support TLS between the components [#904](https://github.com/pingcap/tidb-binlog/pull/904) [#894](https://github.com/pingcap/tidb-binlog/pull/894)
+    + Add the `kafka-client-id` configuration item in Drainer to configure Kafka’s client ID [#902](https://github.com/pingcap/tidb-binlog/pull/902)
+    + Support purging the incremental backup data in Drainer [#885](https://github.com/pingcap/tidb-binlog/pull/885)
+
+* TiDB Ansible
+
+    + Support deploying multiple Grafana/Prometheus/Alertmanagers in one cluster [#1142](https://github.com/pingcap/tidb-ansible/pull/1142)
+    + Add the `metric_port` configuration item (`8234` by default) in TiFlash's configuration file [#1145](https://github.com/pingcap/tidb-ansible/pull/1145)
+    + Add the `flash_proxy_status_port` configuration item (`20292` by default) in TiFlash's configuration file [#1141](https://github.com/pingcap/tidb-ansible/pull/1141)
+    + Add the TiFlash monitoring dashboard [#1147](https://github.com/pingcap/tidb-ansible/pull/1147) [#1151](https://github.com/pingcap/tidb-ansible/pull/1151)
+
+## Bugfix
+
+* TiDB
+
+    + Fix the issue that an error is reported when creating `view` with a column name that exceeds 64 characters [#14850](https://github.com/pingcap/tidb/pull/14850)
+    + Fix the issue that duplicate data occurs in `information_schema.views` because the `create or replace view` statement is incorrectly processed [#14832](https://github.com/pingcap/tidb/pull/14832)
```

Comment:
```suggestion
    + Fix the issue that duplicate data exists in `information_schema.views` because the `create or replace view` statement is incorrectly processed [#14832](https://github.com/pingcap/tidb/pull/14832)
```